db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/DataDictionary.java

```diff
@@ -230,17 +230,15 @@
     int SYSPERMS_CATALOG_NUM=20;
     int SYSUSERS_CATALOG_NUM=21;
     int SYSBACKUP_CATALOG_NUM=22;
-    int SYSBACKUPFILESET_CATALOG_NUM=23;
```

Comment:
is it better to remove these, or leave the integers in the schema as 'unused placeholders', to reduce reverse compatibility problems after an upgrade?

Reply:
Good question. We never dropped system tables before. I'm not aware of any consequences if we modify existing table number. I've run upgrade tests, and it looks good to me.

Reply:
My suggestion is to leave the unchanged numbers alone, i.e. remove '23 and 24', but leave the numbers 'the same' for all the others 25 through 33, so we'd end the list on:
  SYSSNAPSHOT_NUM=32; and SYSTOKENS_NUM=33;

In this way, we wont have a problem accessing any records written to a pre-existing database that has already stored a data value based on the integers 25 through 33.  

Either that, or I am not understanding the data dictionary correctly!

Reply:
Agreed, I think i'm happier with that sort of approach (just to be safe?).